scripts/deploy.ts

```diff
@@ -177,13 +193,73 @@ async function main() {
   )
 
   if (shouldTransferOldAccounts) {
+    const oldKetlAttesationContract = await KetlAttestation__factory.connect(
```

Comment:
Can we also use `console.log` when we start something to do? So the developer won't get confused by an empty console with the final info of the previous operation

Reply:
Resolved in https://github.com/BigWhaleLabs/ketl-attestation-token/pull/33/commits/dced6cef129e7ad14ed1aab240a7f8ca2533e7d8.